src/com/company/Menu.java

```diff
@@ -0,0 +1,37 @@
+package com.company;
+
+public class Menu {
```

Comment:
@SFDC-Melby-Arguedas This is the ContactManager, maybe Menu is not a correct name